app-accessibility/flite/flite-2.1.recipe

```diff
@@ -0,0 +1,142 @@
+	lib:libflite_cmu_us_awb$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_kal$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_kal16$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_rms$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_slt$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmulex$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_usenglish$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	flite${secondaryArchSuffix}_devel = $portVersion
+	devel:libflite$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_grapheme_lang$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_grapheme_lex$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_indic_lang$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_indic_lex$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_time_awb$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_awb$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_kal$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_kal16$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_rms$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_slt$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmulex$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_usenglish$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	flite$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage flite$secondaryArchSuffix \
+	$commandBinDir/flite_cmu_time_awb \
+	$commandBinDir/flite_cmu_us_awb \
+	$commandBinDir/flite_cmu_us_kal \
+	$commandBinDir/flite_cmu_us_kal16 \
+	$commandBinDir/flite_cmu_us_rms \
+	$commandBinDir/flite_cmu_us_slt \
+	$commandBinDir/flite_time \
+	$libDir/libflite.so.$libVersion \
+	$libDir/libflite_cmu_indic_lang.so.$libVersion \
+	$libDir/libflite_cmu_indic_lex.so.$libVersion \
+	$libDir/libflite_cmu_time_awb.so.$libVersion \
+	$libDir/libflite_cmu_us_awb.so.$libVersion \
+	$libDir/libflite_cmu_us_kal16.so.$libVersion \
+	$libDir/libflite_cmu_us_slt.so.$libVersion \
+	$libDir/libflite_cmulex.so.$libVersion \
+	$libDir/libflite_usenglish.so.$libVersion
+
+BUILD()
+{
+	autoconf
+	runConfigure ./configure --enable-shared
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make $jobArgs install
+	prepareInstalledDevelLibs \
+		libflite \
+		libflite_cmu_grapheme_lang \
+		libflite_cmu_grapheme_lex \
+		libflite_cmu_indic_lang \
+		libflite_cmu_indic_lex \
+		libflite_cmu_time_awb \
+		libflite_cmu_us_awb \
+		libflite_cmu_us_kal \
+		libflite_cmu_us_kal16 \
+		libflite_cmu_us_rms \
+		libflite_cmu_us_slt \
+		libflite_cmulex \
+		libflite_usenglish
+	packageEntries devel $developDir
```

Comment:
I'm in favour of removing the binaries from secondary architecture here, they will probably collide as they are both installed in $prefix/bin for gcc2h ...

Reply:
Should I use a case that detects whether the x86_gcc2 architecture is being used and exclude everything from line 127 and below if that's the case?

Reply:
It's a bit more complex now, with the debuginfopackage